src/target/riscv/opcodes.h

```diff
@@ -17,214 +17,163 @@ static uint32_t bit(uint32_t value, unsigned int b)
 	return (value >> b) & 1;
 }
 
+#define INST_RD(r)	(bits(r, 4, 0) << 7)
```

Comment:
Can you make these static functions instead of macros? Modern compilers will optimize regardless, and provide better type checking etc.

Reply:
Done